tests/manage/cluster/nodes/test_az_failure.py

```diff
@@ -0,0 +1,54 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import ManageTest
+from ocs_ci.framework import config
+from ocs_ci.utility import aws
+
+logger = logging.getLogger(__name__)
+
+
+# TODO create Polarion item
+@pytest.mark.polarion_id("")
+@pytest.mark.skipif(
+    condition=config.ENV_DATA['platform'] != 'AWS',
+    reason="Tests are not running on AWS deployed cluster"
+)
+class TestAvailabilityZones(ManageTest):
+    """
+    test availability zone failure:
+    test stages:
+    1. block 1 AZ by attaching security group with no permissions
+    2. validate - cluster functionality and health
+        2a. health check - warning or error
+        2b. create cephfs, create rbd, create pvc (validate_cluster)
+
+    3. restore AZ access
```

Comment:
Step missing no function called to check

Reply:
current pull request contains only test infrastructure without actual test and its tear down, to be done in next PR

Reply:
this is problematic as we'll have test which is not valid.
2 options:
1) the test should be skipped until you take care of all its content
2) add the rest of the content in this PR

Reply:
+1 to Raz' comment
I suggest you add this decorator over the correct scope in order to skip everything for now:

`@pytest.mark.skipif(condition=True, reason="Tests are not ready to be ran")`

Reply:
skip added